modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -185,6 +185,104 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-171793 - Verify it's possible to create a notification action with the trigger On Order Status Update"
+	@priority = "5"
+	test CanCreateActionNotificationWithOnOrderStatusUpdate {
+		property portal.acceptance = "true";
+
+		task ("Given: A notification template is created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test test",
+				name = "Notification Template",
+				subject = "Subject");
+
+			Button.clickSave();
+		}
+
+		task ("When: A notification action with the trigger On Order Status Update is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Commerce Order");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Notification action when order status update",
+				active = "true",
+				inputExpression = "orderStatus == 0",
+				notificationName = "Notification Template",
+				thenAction = "Notification",
+				whenAction = "On Order Status Update");
+		}
+
+		task ("And when: an account, minium site and an order is created") {
```

Comment:
I think it makes more sense to have these as And given. All of these things you are creating is setup.